### ISSUE TITLE
Heading vs Track

### ISSUE THREAD

**Issue by yohanboniface** [UAVTracking/UAVTrackingProtocol#8]
Currently, the protocol contains a `heading` field, but it defines with the definition of `Track`. Which of both should go in the protocol, if any?

Heading: where the aircraft nose is actually pointing at, can be obtained from a compass
Track: actual path of the aircraft over ground, a GPS is needed to obtain this info

![3xqzo](https://user-images.githubusercontent.com/146023/31557720-3aa3b574-b04a-11e7-9cd2-047de50aa29e.jpg)

**Comment by darekzbik** [comment]
It should be 'track' as wind calculation is something what need to be avoided. To void confusion in doc it can be called 'course over ground'.

And yes we need a track to avoid collisions. When you receive a single packet with heading you can calculate a risk of collision immediately. If you try to reconstruct movement based on packets it will be risky as some (lot of) packets can be lost in transmission.

Precision of this field can be lower let's say 1/128 of a circle

**Comment by pjalocha** [comment]
In my specification I call it now "track over ground", in the current software it is indeed not very correctly called "heading". https://docs.google.com/document/d/1lx5d6qcAk7sZULNXXTvC7bQsNCZKk73gKW74jdlIMA4/edit#

**Comment by optimaltracking** [comment]
yes, we will not have compas in "cheap transmitters". so track is the best
